client/app/components/pages/Home.vue

```diff
@@ -31,39 +42,131 @@
 <script>
 
 import Navigation from '../partials/Navigation.vue'
-import GeneCard  from '../viz/GeneCard.vue'
+import GeneCard  from  '../viz/GeneCard.vue'
+import VariantCard    from  '../viz/VariantCard.vue'
 
+import CohortModel    from  '../../models/CohortModel.js'
+import FilterModel    from  '../../models/FilterModel.js'
+import GeneModel      from  '../../models/GeneModel.js'
 
 
 export default {
   name: 'home',
   components: {
       Navigation,
-      GeneCard
+      GeneCard,
+      VariantCard
   },
   props: [],
   data() {
     return {
       greeting: 'gene.iobio.vue',
       selectedGene: {},
       selectedTranscript: {},
-      geneRegionBuffer: 1000
+      geneRegionBuffer: 1000,
+      geneRegionStart: null,
+      geneRegionEnd: null,
+
+
+      cohortModel: null,
+      geneModel: null,
+      filterModel: null
     }
   },
 
   created: function() {
-    genomeBuildHelper.promiseInit({DEFAULT_BUILD: null});
+
+  },
+
+  mounted: function() {
+    let self = this;
+
+
+    genomeBuildHelper.promiseInit({DEFAULT_BUILD: 'GRCh37'})
+    .then(function() {
+      return self.promiseInitCache();
+    })
+    .then(function() {
+      return cacheHelper.promiseClearStaleCache();
+    })
+    .then(function() {
+      // Instantiate helper class than encapsulates IOBIO commands
+      endpoint = new EndpointCmd(useSSL, IOBIO, cacheHelper.launchTimestamp, genomeBuildHelper, utility.getHumanRefNames);
+
+    })
+    .then(function() {
+      self.geneModel = new GeneModel();
+      self.geneModel.geneSource = siteGeneSource;
+
+      self.cohortModel = new CohortModel(self.geneModel);
+      return self.cohortModel.promiseInitDemo();
+    })
+    .then(function() {
+      self.filterModel = new FilterModel(self.cohortModel.affectedInfo);
+    },
+    function(error) {
+
+    })
```

Comment:
I prefer using `.catch(function(error) { })` because you can chain it after `.then`.